### PR TITLE
New script to update version names and codes

### PR DESCRIPTION
**Pull request by maxme**
cc @mzorz 

Note: I'm not a fan of scripts committing anything to git, we could eventually extract all version names and code outside of a branch.

### CODE REVIEW

File: tools/update-release-names.sh [hunk 1]
```diff
@@ -1,8 +1,9 @@
 #!/bin/sh
 
 if [ x"$4" == x ]; then
-  echo "Usage:   $0 version-code current-version beta-version alpha-version"
+  echo "Usage:   $0 version-code current-version beta-version alpha-version [current-branch] [beta-branch] [alpha-branch]"
```
Comment (mzorz) [inline tools/update-release-names.sh]: Should we update the `version-code` parameter naming here for `current-alpha-version-code` to make sure the people who run this script ;) use the right starting version code number? And then let the script make that number plus one. I think it's easier to just look up for the version code in `alpha` and let the script generate the next 3 codes than having the user have to remember they need to look up and then add one more. Maybe an automated check would be ideal (pull latest alpha, check versionCode....), but that's too much to ask, good for a trial though

Comment (maxme) [reply tools/update-release-names.sh]: Good idea, but I think I'll do something differently: I'll check the version code in `develop` and compare with the one entered by the user, if it's not greater than the one we have in `develop` I'll make the script fail.

Comment (mzorz) [reply tools/update-release-names.sh]: 💯

Comment (maxme) [reply tools/update-release-names.sh]: Updated in 0207ce2

**Comment by mzorz** [comment]
:shipit:

**Pull request merged**